### ISSUE TITLE
Tag the source

### ISSUE THREAD

**Issue by fabaff** [LeandroBarone/python-ascii_magic#10]
It would be very helpful if you could tag releases as well. This would enable distributions who want to run your tests to fetch the package from GitHub instead of PyPI, where the tests are excluded.

Thanks

**Comment by LeandroBarone** [comment]
Good point, can you confirm that the releases/tags I've just created are correct? Thanks!

**Comment by fabaff** [comment]
The release on PyPI is [2.1.1](https://pypi.org/project/ascii-magic/2.1.1/), the tag says [2.1](https://github.com/LeandroBarone/python-ascii_magic/releases/tag/v2.1).

There is also a difference between what is in the published tarball on PyPI and in the repo. The image files for the tests are missing in PyPI tarball. The other way around is `setup.py` and `setup.cfg` in the tarball but not the repo.

**Comment by LeandroBarone** [comment]
I've added the tag 2.1.1 and created a new tag v2.2 for the latest tarball on PyPI, which includes the image files for the tests.
